ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -753,7 +753,9 @@ export class ObjectDetailComponent implements OnInit {
         if (res === '') {
           this.isObjectSelected = false
         }
+        this.data.getConversionRate()
```

Comment:
The status for restored table still doesn't appear : https://drive.google.com/file/d/1AMTWnBV6dYVQsxRhDFKlAk3e35GRO5tF/view?usp=sharing&resourcekey=0-QqH1NiMA_PJNztN45FXxYQ